docs/api-reference/core/transform.md

```diff
@@ -192,7 +192,10 @@ Constructs a `Transform` object. It then creates destination buffers if needed a
 * `gl` (`WebGL2RenderingContext`) gl - context
 * `opts` (`Object`={}) - options
   * `sourceBuffers` (`Object`) - key and value pairs, where key is the name of vertex shader attribute and value is the corresponding `Attribute`, `Buffer` or attribute descriptor object.
-  * `feedbackBuffers` (`Object`, Optional) - key and value pairs, where key is the name of vertex shader attribute and value is the corresponding `Buffer` object.
+  * `feedbackBuffers` (`Object`, Optional) - key and value pairs, where key is the name of vertex shader varying and value is the corresponding `Buffer` object or buffer params object. If a buffer params object is specified, it will contain following fields, these can be used to capture data into the buffer a particular offset and size.
+    * `buffer`=(Buffer) - Buffer object to be bound.
+    * `offsetInBytes`=(Number, default: 0) - Byte offset that is used to start recording the data in the buffer.
```

Comment:
Is there any way we can reuse the `Accessor` class here (or if not, at least the field naming from that class) instead of defining a new "arbitrary" object/fields?

The names in question are probably `byteLength`, `byteOffset` etc, take a look at luma and glTF accessor definitions, if interested we can review this together.

I think that having a consistent (glTF "compatible") way to specify these things would make it a little easier to remember and use.